README.md

```diff
@@ -1,44 +1,60 @@
 [![Build Status](https://travis-ci.org/cloudfoundry-incubator/eirini.svg?branch=master)](https://travis-ci.org/cloudfoundry-incubator/eirini)
 
-# Eirini - what this?
+# What is Eirini?
 
-Eirini is a Kubernetes backend for Cloud Foundry. It deploys CF apps to a kube backend, using OCI images and Kube deployments.
+*Eirini* is a Kubernetes backend for Cloud Foundry.
+It deploys CF apps to a kube backend, using OCI images and Kube deployments.
 
-_But there's more!_
+Eirini exports staged CF images as docker images.
+So you can schedule them however you'd like.
+*And separately* it gives you the nice integrated `cf push` flow,
+with CF Apps mapped directly to kube deployment objects (pods).
```

Comment:
Currently, we actually map each CF app to a `StatefulSet`.

Reply:
fixed.